extension/scripts/addFileStubsIfNeeded.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+set -euo pipefail
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+PROJECT_DIR="$SCRIPT_DIR/.."
+
+if [ ! -f "$PROJECT_DIR/config.json" ]; then
+  echo {} >"$PROJECT_DIR/config.json"
+fi
+
+if [ ! -f "$PROJECT_DIR/build/blsNetworksConfig.json" ]; then
+  mkdir -p "$PROJECT_DIR/build"
+  echo {} >"$PROJECT_DIR/build/blsNetworksConfig.json"
+fi
```

Comment:
Yeah sounds good to me. TypeScript with ts-node might be worth considering too.

Reply:
This will no longer be needed with https://github.com/web3well/bls-wallet/issues/299 , so we can just remove there. Don't expect this will need to be maintained medium/long term.